packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/trees/TreeItemContextMenuControllerTests.java

```diff
@@ -161,20 +169,15 @@ public void givenPapayaProjectWhenTheContextMenuActionsAreRequestedOnAnObjectThe
     @DisplayName("Given a studio, when the context menu actions are requested on an Entity, then the correct actions are returned")
     public void givenAStudioWhenTheContextMenuActionsAreRequestedOnAnEntityThenTheCorrectActionsAreReturned() {
         // 1- retrieve the tree description id of the DSL Domain explorer example
-        var explorerDescriptionId = new AtomicReference<String>();
-
-        Map<String, Object> explorerVariables = Map.of(
-                "editingContextId", StudioIdentifiers.SAMPLE_STUDIO_EDITING_CONTEXT_ID.toString()
-        );
+        Map<String, Object> explorerVariables = Map.of("editingContextId", StudioIdentifiers.SAMPLE_STUDIO_EDITING_CONTEXT_ID);
```

Comment:
I reworked a bit this test to make checkstyle happy.
If needed I can rework it more to split it into smaller pieces